articles/sorting-algorithms.md

```diff
@@ -0,0 +1,145 @@
+# Sorting algorithms visualised
+- 19th December 2016
+
+Sometimes you or a computer needs to sort a list of items—say, a list of names into alphabetical order, or a list of objects into order of length. To sort this list, especially if it is a large list, you need some kind of process: for example, if you were sorting a list of names into alphabetic order, you could put all the names beginning with A together, then all the Bs together, and then you could sort them by the second letter: all the AAs together, then all the ABs together, until they're sorted.
```

Comment:
"items - say a list of names..."
"some kind of process - for example, if..."
Split last line in two - "Then all the Bs together. Then you'd sort them by the second letter - all the AAs, ABs, ACs, until all names are sorted."